backend/endpoints/v2/index.ts

```diff
@@ -59,12 +59,10 @@ const AddEvents = Endpoint({
 
 const GetChannelADVStats = Endpoint({
   Method: "GET",
-  getPath: ({ channelId, sdate, edate }) => `/v2/ad/channel/${channelId}/since-${sdate}/till-${edate}`,
+  getPath: ({ channelId }) => `/v2/ad/channel/${channelId}`,
   Input: {
     Params: t.type({
-      channelId: t.string,
-      sdate: t.string,
-      edate: t.string,
```

Comment:
by removing these props from here, you need to add to `Query` object inside `Input` (at the same level as `Params`) @vecna